product_contract_transmit_method/models/sale_order.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2020 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, models
+
+
+class SaleOrder(models.Model):
+
+    _inherit = "sale.order"
+
+    @api.multi
+    def _prepare_contract_value(self, contract_template):
+        contract_values = super()._prepare_contract_value(contract_template)
+        contract_values["transmit_method_id"] = self.transmit_method_id.id
```

Comment:
usually the simple quotas are used for fields, like so `'transmit_method_id'`

Reply:
It's 13.0 (Black) coding convention (easier to forward-port).